### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by ubald**

### CODE REVIEW

**Comment by ubald** [comment]
[API libs have no license file](https://trello.com/c/5BIM6244/6039-api-libs-have-no-license-file)

**Review by alavers — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Copyright (c) 2015 Smooch Technologies Inc. All rights reserved.
```
Comment (alavers) [inline LICENSE]: Should that be 2018? 🤔

Comment (alavers) [reply LICENSE]: Unless you're basing this off of the initial commit date

Comment (ubald) [reply LICENSE]: I think I copied this from another similar repo that already had a license. Will change.

**Pull request merged**